tfe/resource_tfe_workspace.go

```diff
@@ -46,6 +46,12 @@ func resourceTFEWorkspace() *schema.Resource {
 				ForceNew: true,
 			},
 
+			"description": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Default:  "",
```

Comment:
Sorry, I know we already changed this from `false` before....I don't believe this is necessary at all, right? Descriptions can be `null`; the zero value of the field here will naturally be `""` anyway.